core/string/translation.cpp

```diff
@@ -1316,6 +1681,26 @@ void TranslationServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("clear"), &TranslationServer::clear);
 
 	ClassDB::bind_method(D_METHOD("get_loaded_locales"), &TranslationServer::get_loaded_locales);
+
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_enabled"), &TranslationServer::is_pseudolocalization_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_enabled", "enabled"), &TranslationServer::set_pseudolocalization_enabled);
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_accents_enabled"), &TranslationServer::is_pseudolocalization_accents_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_accents_enabled", "enabled"), &TranslationServer::set_pseudolocalization_accents_enabled);
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_double_vowels_enabled"), &TranslationServer::is_pseudolocalization_double_vowels_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_double_vowels_enabled", "enabled"), &TranslationServer::set_pseudolocalization_double_vowels_enabled);
+	ClassDB::bind_method(D_METHOD("get_pseudolocalization_expansion_ratio"), &TranslationServer::get_pseudolocalization_expansion_ratio);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_expansion_ratio", "p_expansion_ratio"), &TranslationServer::set_pseudolocalization_expansion_ratio);
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_fake_bidi_enabled"), &TranslationServer::is_pseudolocalization_fake_bidi_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_fake_bidi_enabled", "enabled"), &TranslationServer::set_pseudolocalization_fake_bidi_enabled);
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_override_enabled"), &TranslationServer::is_pseudolocalization_override_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_override_enabled", "enabled"), &TranslationServer::set_pseudolocalization_override_enabled);
+	ClassDB::bind_method(D_METHOD("is_pseudolocalization_skip_placeholders_enabled"), &TranslationServer::is_pseudolocalization_skip_placeholders_enabled);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_skip_placeholders_enabled", "enabled"), &TranslationServer::set_pseudolocalization_skip_placeholders_enabled);
+	ClassDB::bind_method(D_METHOD("get_pseudolocalization_prefix"), &TranslationServer::get_pseudolocalization_prefix);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_prefix", "prefix"), &TranslationServer::set_pseudolocalization_prefix);
+	ClassDB::bind_method(D_METHOD("get_pseudolocalization_suffix"), &TranslationServer::get_pseudolocalization_suffix);
+	ClassDB::bind_method(D_METHOD("set_pseudolocalization_suffix", "suffix"), &TranslationServer::set_pseudolocalization_suffix);
```

Comment:
I think this might be a bit overkill in terms of runtime customization (though it might be useful for some plugins I guess). You can leave it in for now and we'll see later on, this is easy to change without impacting the rest much.

Since all the config is exposed in the project settings, we could instead have users set the project settings at runtime, and since their values are cached in the constructor, expose a method to force reloading the pseudolocalization configuration (i.e. reparse all settings).

Reply:
Hey! Should I do this now? I should remove all these functions and replace them with a `reload_pseudolocalization()` method that reparses the settings and reloads the translations, correct?

Reply:
That's up for discussion, what do you think about it yourself in terms of usability?

But yeah, personally I think it would be good to avoid having too many methods dedicated to customizing this at runtime, which is a fairly niche use case.

Reply:
Yeah. I guess we could keep in two methods for toggling pseudolocalization and if someone wants to change other pseudolocalization settings, they could do it by directly changing the project settings and then calling `reload_pseudolocalization()`